multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/steps/CollectSystemParametersStep.java

```diff
@@ -111,6 +113,8 @@ private SystemParameters createSystemParameters(ProcessContext context, CloudCon
                                              .reserveTemporaryRoutes(reserveTemporaryRoutes)
                                              .credentialsGenerator(credentialsGeneratorSupplier.get())
                                              .timestampSupplier(timestampSupplier)
+                                             .namespace(namespace)
+                                             .applyNamespace(applyNamespace)
```

Comment:
Do we really need namespace and applyNamespace in the Builder class? Is it possible to have a method in CollectSystemParametersStep which creates HostValidator and pass it to the builder? This way the builder won't need to create the HostValidator it will be injected instead.